components/FeatureHighlight/examples/FeatureHighlightShownViewExample.m

```diff
@@ -32,17 +32,20 @@ - (id)init {
   return self;
 }
 
-- (void)didTapButton:(id)sender {
-  MDCButtonScheme *buttonScheme = [[MDCButtonScheme alloc] init];
-  buttonScheme.colorScheme = self.colorScheme;
-  buttonScheme.typographyScheme = self.typographyScheme;
+- (MDCContainerScheme *)containerScheme {
+  MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
+  scheme.colorScheme = self.colorScheme;
+  scheme.typographyScheme = self.typographyScheme;
+  return scheme;
+}
 
+- (void)didTapButton:(id)sender {
   MDCFloatingButton *fab = [[MDCFloatingButton alloc] init];
   [fab setImage:[UIImage imageNamed:@"Plus"] forState:UIControlStateNormal];
   [fab sizeToFit];
   fab.center = _button.center;
 
-  [MDCFloatingActionButtonThemer applyScheme:buttonScheme toButton:fab];
+  [fab applySecondaryThemeWithScheme:self.containerScheme];
```

Comment:
Good catch and sorry about the confusion here. The reason `self.` is used here is becauuse `@property(nonatomic, readonly, strong) MDCContainerScheme *containerScheme;` is defined in `FeatureHighlightExampleSupplemental.h`. And the reason I created this property was because a category needs to call `containerScheme` and I don't feel it is good to expose the containerScheme as a method.

Reply:
oh ok, that makes sense